src/lib/components/layout/index.scss

```diff
@@ -149,29 +151,20 @@
 }
 
 .q-layout > .q-drawer {
-  //TODO Problems with z-index between multiple layouts
   z-index: 101;
   border-radius: 0;
 
-  &.above {
+  &--above {
     z-index: 104;
   }
 
-  &.border-radius {
-    &__left {
-      &--top {
-        border-top-left-radius: 16px;
-      }
-      &--bottom {
-        border-bottom-left-radius: 16px;
-      }
-    }
-    &__right {
-      &--top {
-        border-top-right-radius: 16px;
-      }
-      &--bottom {
-        border-bottom-right-radius: 16px;
+  $horizontal: "left", "right";
+  $vertical: "top", "bottom";
+
+  @each $vPos in $vertical {
```

Comment:
Same here, also `$verticalPos` or `$vertical-pos` would be a bit more descriptive. Not a blocker, just a minor comment.